src/languages/mailmojo-nb_NO.po

```diff
@@ -13,154 +13,164 @@ msgstr ""
+"\">MailMojo settings page</a> and make sure the access token is correct."
+msgstr ""
+"Dine epostlister er ikke tilgjengelig. Vennligst gå til <a href=\"%s"
+"\">MailMojo-innstillinger</a> og sørg for at tilgangsnøkkelen er riktig."
+
+#: mailmojo-widget.php:97
+msgid ""
+"You don't have any email lists in MailMojo. Please go to MailMojo and add "
+"one. Refresh this page when done."
 msgstr ""
-"Du må fylle ut din MailMojo-kontoinformasjon under <a href=\"%s"
-"\">innstillinger for MailMojo</a>"
+"Du har ingen epostlister i MailMojo. Vennligst gå til MailMojo og legg til "
+"dette. Oppdater denne siden når det er gjort."
 
-#: mailmojo-widget.php:64
+#: mailmojo-widget.php:105
 msgid "Newsletter Signup"
 msgstr "Nyhetsbrev"
 
-#: mailmojo-widget.php:67
-msgid "Interests"
-msgstr "Interesser"
+#: mailmojo-widget.php:108
+msgid "Interests:"
+msgstr "Interesser:"
 
-#: mailmojo-widget.php:69
+#: mailmojo-widget.php:112
 msgid "Sign me up!"
 msgstr "Meld meg på!"
 
-#: mailmojo-widget.php:119
-msgid "General"
-msgstr "Generelt"
+#: templates/widget-admin.php:3
+msgid "Subscribe to Email List"
+msgstr "Påmeld til epostlisten"
 
-#: mailmojo-widget.php:120
-msgid "MailMojo List ID"
-msgstr "MailMojo liste-ID"
-
-#: mailmojo-widget.php:121
-msgid ""
-"To find the list ID: Go to the email list of your choice in MailMojo, and "
-"look at the last part of the URL. That is the list ID. E.g. given \"mailmojo."
-"no/lists/123\", 123 is the list ID."
-msgstr ""
-"For å finne liste-IDen kan du gå til en av dine e-postlister i MailMojo, og "
-"se på den siste delen av URLen. Dette tallet er liste-IDen. F.eks. med "
-"«mailmojo.no/lists/123» er 123 liste-IDen. "
-
-#: mailmojo-widget.php:122
+#: templates/widget-admin.php:17
 msgid "Title"
 msgstr "Tittel"
 
-#: mailmojo-widget.php:123
+#: templates/widget-admin.php:26
 msgid "Description Below Title"
 msgstr "Beskrivelse under tittelen"
 
-#: mailmojo-widget.php:124
-msgid "Include name field"
-msgstr "Inkluder navnfelt"
-
-#: mailmojo-widget.php:125
+#: templates/widget-admin.php:34
 msgid "Signup Button Text"
 msgstr "Påmeldingsknapp"
 
-#: mailmojo-widget.php:126
-msgid "Optional Tags"
-msgstr "Valgfrie merkelapper"
+#: templates/widget-admin.php:47
+msgid "Include name field"
+msgstr "Inkluder navnfelt"
 
-#: mailmojo-widget.php:127
-msgid "Tag Selection Label"
-msgstr "Overskrift for merkelapper"
+#: templates/widget-admin.php:52
+msgid "Selectable tags"
+msgstr "Merkelapper som kan velges"
 
-#: mailmojo-widget.php:128
-msgid "Tags (comma separated)"
-msgstr "Merkelapper (kommaseparert)"
+#: templates/widget-admin.php:53 templates/widget-admin.php:93
+msgid "optional"
+msgstr "valgfri"
 
-#: mailmojo-widget.php:169
-msgid "Name"
-msgstr "Navn"
+#: templates/widget-admin.php:57
+msgid "Tag selection label"
+msgstr "Overskrift for valgfri merkelapper"
 
-#: mailmojo-widget.php:202
-msgid "E-mail"
-msgstr "E-postadresse"
+#: templates/widget-admin.php:65
+msgid "Type"
+msgstr "Type"
```

Comment:
"Valgmuligheter" in Norwegian, and then change the English to "Selection options".